compiler/ras/Tree.cpp

```diff
@@ -2865,6 +2865,7 @@ int32_t childTypes[] =
    TR::Address,                    // TR::checkcast
    TR::Address,                    // TR::checkcastAndNULLCHK
    TR::Address,                    // TR::New
+   TR::NoType | (TR::Address<<8),  // TR::newvalue
```

Comment:
I'm not convinced this is the right answer here. The array represents the child types and unless I'm mistaken a newvalue has one child and it's type is TR:Address. So why not have that as the entry in this childTypes table (just as TR::New does) ?

Reply:
This was discussed in a previous comment, see https://github.com/eclipse/omr/pull/4156#discussion_r308031914.

`newValue` actually takes an arbitrary number of children greater than one. The first child is used to indicate the type of the object produced and the other children specify the values used to initialize the object.